.gitlab-ci.yml

```diff
@@ -368,3 +369,15 @@ deploy:
       - SCRIPT: install_script_vector0.sh
   script:
     - $S3_CP_CMD ./${SCRIPT} s3://dd-agent/scripts/${SCRIPT} --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers full=id=3a6e02b08553fd157ae3fb918945dd1eaae5a1aa818940381ef07a430cf25732
+  after_script:
+    # invalidate the install.datadoghq.com CF distribution
+    - export $(printf "AWS_ACCESS_KEY_ID=%s AWS_SECRET_ACCESS_KEY=%s AWS_SESSION_TOKEN=%s"
+        $(aws --region "us-east-1" sts assume-role
+          --duration-seconds 900
+          --role-arn "arn:aws:iam::464622532012:role/build-stable-cloudfront-invalidation"
+          --role-session-name "build-stable-cloudfront-invalidate-script"
+          --query "Credentials.[AccessKeyId,SecretAccessKey,SessionToken]"
+          --output text
+        )
+      )
+    - aws --region "us-east-1" cloudfront create-invalidation --distribution-id "E2VSER0FO39KRV" --paths "/scripts/*"
```

Comment:
This looks good, although out of curiosity, can't the same be achieved in a simpler way with `aws-vault`?

Reply:
It can be achieved with aws-vault, but then you would have to create vault config file in a way very similar to how we create the environment variables, so basically there would be little difference.